new.go

```diff
@@ -281,17 +282,18 @@ func newBuilder(ctx context.Context, store storage.Store, options BuilderOptions
 	coptions := storage.ContainerOptions{}
 	coptions.IDMappingOptions = newContainerIDMappingOptions(options.IDMappingOptions)
 
+	attempts := 0
+	name = fmt.Sprintf("%s-%d", name, rand.Int())
 	container, err := store.CreateContainer("", []string{name}, imageID, "", "", &coptions)
-	suffix := 1
 	for err != nil && errors.Cause(err) == storage.ErrDuplicateName && options.Container == "" {
-		suffix++
-		tmpName := fmt.Sprintf("%s-%d", name, suffix)
-		if container, err = store.CreateContainer("", []string{tmpName}, imageID, "", "", &coptions); err == nil {
-			name = tmpName
+		attempts++
+		name = fmt.Sprintf("%s-%d", name, rand.Int())
+		if container, err = store.CreateContainer("", []string{name}, imageID, "", "", &coptions); err == nil {
+			break
+		}
+		if attempts == 100 {
```

Comment:
Could we change this to '> 99', just to be ultra safe.

Reply:
Will change.